examples/llm/src/models/mosaic_gpt.py

```diff
@@ -99,6 +99,17 @@ def __init__(self, cfg: DictConfig):
         else:
             self.attn_mask = None
 
+        if cfg.get('no_bias', False):
+            for module in self.modules():
+                if hasattr(module, 'bias') and isinstance(
+                        module.bias, nn.Parameter):
+                    if cfg.get('verbose'):
+                        print(f'Removing bias ({module.bias}) from {module}.')
+                    module.register_parameter('bias', None)
+
+        if cfg.get('verbose') and cfg.get('verbose') > 2:
```

Comment:
Should it be >= 2? What secret behavior does `cfg.verbose = 2` do? 👀

Reply:
This is me just "thinking out loud" from my undergrad days:
verbosity levels add more and more verbosity. Printing the whole model is on the extreme side of verbose, so I set it to only print if someone sets the verbose flag to >2.

Reply:
Since we're adding a verbose flag, might as well add levels  🤷‍♂️ 
or i can just rm the model print and we can default to using `verbose` as purely a bool everywhere.